docs/tools/firmware.sh

```diff
@@ -132,59 +129,58 @@ create_rpm () {
 	fi
 
 	# Create the spec file
-	echo "Name:       apple-firmware" > $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "Version:    ${ver}" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "Release:    1" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "Summary:    Wi-Fi and Bluetooth firmware for T2 Macs" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "License:    Proprietary" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-
-	echo -e "\n%description" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "Wi-Fi and Bluetooth firmware for T2 Macs" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-
-	echo -e "\n%install" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "mkdir -p %{buildroot}/usr/lib/firmware/brcm" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "install -m 644 * %{buildroot}/usr/lib/firmware/brcm" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-
-	echo -e "\n%post" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "modprobe -r brcmfmac_wcc || true" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "modprobe -r brcmfmac || true" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "modprobe brcmfmac || true" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "modprobe -r hci_bcm4377 || true" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "modprobe hci_bcm4377 || true" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-
-	echo -e "\n%files" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "/usr/lib/firmware/brcm/*" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-
-	# Set OS to Linux
-	if [ -f "$HOME/.rpmrc" ]
-	then
-		echo -e "\nBacking up existing .rpmrc"
-		cp ${verbose} $HOME/.rpmrc $HOME/.rpmrc_orj
-	fi
-	echo "buildostranslate: Darwin: Linux" > $HOME/.rpmrc
+	cat <<- EOF > $HOME/rpmbuild/SPECS/apple-firmware.spec
+		Name:       apple-firmware
+		Version:    ${ver}
+		Release:    1
+		Summary:    Wi-Fi and Bluetooth firmware for T2 Macs
+		License:    Proprietary
+		BuildArch: noarch
+
+		Source1: firmware.tar
+		Source2: brcmfmac4364b2-pcie.txt
```

Comment:
Source2 and Source3 shall not always be present. Won't appending to the tar file be a better option?